packages/data-portal-explore/src/lib/dataTableHelpers.tsx

```diff
@@ -147,7 +147,7 @@ export function truncatedTableCell<T>(cellData: T) {
     const value = _.isFunction(selector)
         ? selector(cellData)
         : cellData[selector as keyof T];
-
+        
```

Comment:
Looks like this file is not completely reverted. You can probably use some git revert / reset command.